### PR TITLE
Replace old docker compose filename

### PR DESCRIPTION
**Pull request by Xbirdfr**
Replace old docker compose filename from

docker-compose.yml/yaml 

to 

compose.yaml

### CODE REVIEW

**Review by maxhelias — APPROVED** [review]

File: .doctor-rst.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ whitelist:
         - '/``.yml``/'
         - '/(.*)\.orm\.yml/' # currently DoctrineBundle only supports .yml
         - /docker-compose\.yml/
```
Comment (OskarStark) [inline .doctor-rst.yaml:95]: this can be removed then

File: .doctor-rst.yaml [hunk 2]
```diff
@@ -93,6 +93,7 @@ whitelist:
...
+        - /compose\.yaml/
```
Comment (OskarStark) [inline .doctor-rst.yaml:96]: this should not be needed

Comment (Xbirdfr) [reply .doctor-rst.yaml:96]: i'm not aware of what this file is doing, can you explain it to me ?

**Review by wouterj — APPROVED** [review]

File: .doctor-rst.yaml (hunk 2 above)
Comment (OskarStark) [reply .doctor-rst.yaml:96]: Sure this list is to suppress a warning by out linter, but as this line will not produce this error anymore it can be removed

**Comment by javiereguiluz** [comment]
@Xbirdfr thanks ... and congrats on your first Symfony Docs contribution 🎉

**Comment by javiereguiluz** [comment]
Note: I merged this in 7.1 because I missed that reviewers suggested to merge this in 5.4 and up. But, later, I backported these changes to 5.4 and all the other branches. Thanks.

**Pull request merged**